polystores/stores/s3_store.py

```diff
@@ -70,6 +78,10 @@ def set_env_vars(self):
             os.environ['AWS_SECURITY_TOKEN'] = self._aws_session_token
         if self._region_name:
             os.environ['AWS_REGION'] = self._region_name
+        if self._aws_use_ssl:
```

Comment:
What happend when this is `false`, should we test for `None` instead?

Reply:
Sorry missed that one! Will provide update!

Reply:
Fixed!